src/main/java/nextstep/subway/station/application/StationService.java

```diff
@@ -24,6 +32,14 @@ public StationResponse saveStation(StationRequest stationRequest) {
         return StationResponse.of(persistStation);
     }
 
+    @Transactional(readOnly = true)
+    public Station findByIdThrow(Long id) {
+        Station station = stationRepository.findById(id)
+                                           .orElseThrow(
+                                               () -> new StationNotFoundException(DB_ERROR, "지하철역이 존재하지 않습니다."));
+        return station;
```

Comment:
inline으로 바꾸어도 괜찮을 것 같습니다 :)

추가적으로 이전에 드린 custom exception에 관한 글을 읽어보시고 예외에서 메시지를 함께 관리하는건 어떨까요?
https://github.com/next-step/atdd-subway-admin/pull/443#discussion_r754436515

```suggestion
        return stationRepository.findById(id)
                .orElseThrow(() -> new StationNotFoundException(DB_ERROR, "지하철역이 존재하지 않습니다."));
```